packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.html

```diff
@@ -214,12 +214,22 @@
                             ></dt>
                             <dd
                                 class="oui-tile__description"
-                                data-translate="{{'server_datacenter_' + $ctrl.server.datacenter.split('_')[0]}}"
-                                data-translate-values="{
-                                    number: $ctrl.server.datacenter.split('_')[1]
-                                  }"
-                            ></dd>
+                            >
+                                <span data-ng-bind=":: $ctrl.server.datacenter.toUpperCase().split('_').join('') + ' - ' + ( 'server_datacenter_' + $ctrl.server.datacenter.toUpperCase().split('_')[0] | translate )"
```

Comment:
```suggestion
                                <span data-ng-bind=":: $ctrl.server.iso + ' - ' + ( 'server_datacenter_' + $ctrl.server.iso | translate )"
```

Reply:
`$ctrl.server.datacenter.toUpperCase().split('_').join('')` -> rbx8

`'server_datacenter_' + $ctrl.server.datacenter.toUpperCase().split('_')[0]` -> server_datacenter_rbx
`server_datacenter_rbx | translate` -> Roubaix

`$ctrl.server.iso` -> FR
`'server_datacenter_' + $ctrl.server.is` -> server_datacenter_FR -> translation doesn't exist